tests/unit/test_files.py

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2012-2022 Snowflake Computing Inc. All rights reserved.
+#
+import pytest
+
+from snowflake.snowpark.files import SnowflakeFile
+
+
+def test_create_snowflakefile():
+    snowflake_file = SnowflakeFile.open("test_file_location")
```

Comment:
Does open and not close a file leak any resources?

Reply:
fixed.